app/calculator.js

```diff
@@ -0,0 +1,63 @@
+/**
+ *
+ * A simple calss which performs the sumarize and multiply actions
+ * @class Calculator
+ */
+class Calculator {
+  /**
+   * Creates an instance of Calculator.
+   * @param {*} number is the first incoming value for the calculations
+   * @param {*} increment is the second incoming value for summarize action
+   * @param {*} multiplier is the second incoming value for the multiply action
+   * @memberof Calculator
+   */
+  constructor(number, increment, multiplier) {
```

Comment:
What is the use of these parameters if they are not used anywhere in the methods?